### PR TITLE
Add mehod->method

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

**Review by peternewman — APPROVED** [review]
LGTM

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -18258,6 +18258,7 @@ meerkrat->meerkat
 meetign->meeting
 meganism->mechanism
 mege->merge
+mehod->method
 mehods->methods
 mehtod->method
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:18263]: You could also add the methodical variant of these if you wanted.

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: Added variants 8bfad59

**Review by peternewman — APPROVED** [review]
LGTM thanks.

**Pull request merged**